src/apdu_constants.h

```diff
@@ -51,75 +63,70 @@
 
 #endif
 
-#define OFFSET_CLA   0
-#define OFFSET_INS   1
-#define OFFSET_P1    2
-#define OFFSET_P2    3
-#define OFFSET_LC    4
-#define OFFSET_CDATA 5
+enum { OFFSET_CLA = 0, OFFSET_INS, OFFSET_P1, OFFSET_P2, OFFSET_LC, OFFSET_CDATA };
```

Comment:
Really not a fan of single line enums
Can we change the clang-format conf ?

Reply:
This will be looked into in another PR to find a better configuration.